app/mailers/planning_application_mailer.rb

```diff
@@ -26,6 +26,17 @@ def validation_notice_mail(planning_application, host)
     )
   end
 
+  def receipt_notice_mail(planning_application, host)
+    @host = host
+    @planning_application = planning_application
+
+    view_mail(
+      NOTIFY_TEMPLATE_ID,
+      subject: "We have received your application",
+      to: @planning_application.applicant_email,
```

Comment:
If it we don't have an applicant email, but do have an agent email, should we not send that to them?

Reply:
This is the current methodology for all correspondence. We'll need a new ticket that looks at both applicant and agent details and makes a decision.